event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.ps1

```diff
@@ -4,19 +4,22 @@ $storageName = "contosostorage"
 # Provide an endpoint for handling the events.
 $myEndpoint = "<endpoint URL>"
 
+# Provide the name of the resource group to contain the storage account.
+$myResourceGroup="<resource group name>"
+
 # Create resource group
-New-AzureRmResourceGroup -Name myResourceGroup -Location westus2
+New-AzureRmResourceGroup -Name $myResourceGroup -Location westus2
 
 # Create the Blob storage account. 
-New-AzureRmStorageAccount -ResourceGroupName myResourceGroup `
+New-AzureRmStorageAccount -ResourceGroupName $myResourceGroup `
```

Comment:
Not fault-tolerant for if the named storage account already exists. It's not clear how fault-tolerant our samples need to be.

(cc @erickson-doug on this one because it's actually an important part of our samples strategy.)

Reply:
removed hard-coded name